frontend/src/app/active-sessions/active-sessions.component.css

```diff
@@ -3,55 +3,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-mat-card {
-  width: fit-content;
-  margin: 10px;
-}
-
-button {
-  margin: 10px 5px 0;
-}
-
-.flexbox {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-
-.wrapper {
-  display: flex;
-  flex-direction: column;
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
-  align-items: center;
-}
-.header {
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 10px;
-}
-
-#title {
-  font-size: 24px;
-  text-align: center;
-}
-
-h2 {
-  margin: 5px 10px;
-}
-
-.header > .type {
-  background-color: var(--primary-color);
-  color: white;
-  border-radius: 5px 5px 0 0;
-}
-
-.header > .state {
-  color: white;
-  border-radius: 0 0 10px 10px;
-}
-
```

Comment:
The removal is intentional. In this case the styling was interfering with the default styles provided by MaterialUI. I think it's better to use the Material styling component instead of hacking around them.

Reply:
> In this case the styling was interfering with the default styles provided by MaterialUI. 
What was interfering? Styling the components in CSS is used in the general Angular Material examples. As long as we don't touch the internally used classes of Angular Material, I don't see a problem here. 

> I think it's better to use the Material styling component instead of hacking around them.
Which Material Styling component do you mean? Writing a custom [Angular Material theme](https://material.angular.io/guide/theming-your-components)?

Reply:
If you're talking about global styles, there was some progress to make the style a bit more consistent across the platform (not merged yet, but you can see it here): https://github.com/DSD-DBS/capella-collab-manager/blob/v2/t4c-instances/frontend/src/styles.css